app/models/miq_server.rb

```diff
@@ -46,7 +46,7 @@ class MiqServer < ActiveRecord::Base
   STATUSES_ACTIVE  = [STATUS_STARTING, STATUS_STARTED]
   STATUSES_ALIVE   = STATUSES_ACTIVE + [STATUS_RESTARTING, STATUS_QUIESCE]
 
-  RESTART_SCRIPT = Rails.root.join("vmdb_restart").freeze
+  RESTART_EXIT_STATUS = 123
```

Comment:
Not sure it really helps, but: http://stackoverflow.com/questions/1101957/are-there-any-standard-exit-status-codes-in-linux